### PR TITLE
Make NFS use static ports and document the ports that need to be publ…

### LINKED ISSUE #9: TCP & UDP ports required
NFS clients might use TCP or UDP depending upon their version. More ports need to be published to account for this than are hinted at by the EXPOSE command, and ports for statd, mountd and nlock are also needed.

### PR DESCRIPTION
**Pull request by realflash**
…ished

Fixes #9 . This takes me to a working container. Thanks for creating this. I'm going to diverge now and make it do something slightly different for my purposes, so I probably won't send any more.

### CODE REVIEW

**Comment by realflash** [comment]
Hold on, I'll merge your stuff to make the pull work.

**Comment by eduardocardoso** [comment]
Ok, thanks for all your input. 
Once there are no conflicts I'll merge this.

File: README.md [hunk 1]
```diff
@@ -9,24 +9,35 @@ If you don't want to use docker to achieve this, just look at _run.sh_ and steal
 2. Download [automated build](https://registry.hub.docker.com/u/ecardoso/s3fs-nfs/) from public [Docker Hub Registry](https://registry.hub.docker.com/): `docker pull ecardoso/s3fs-nfs`
 
    (alternatively, you can build it yourself: `docker build -t="ecardoso/s3fs-nfs" github.com/eduardocardoso/s3fs-container`)
-4. On Linux machines you need to load the NFS kernel modules. Add the following to the bottom of _/etc/modules_ and reboot:
+3. On Linux machines you need to load the NFS kernel modules. Add the following to the bottom of _/etc/modules_:
+```
+nfs
+nfsd
+```
```
Comment (eduardocardoso) [inline README.md]: If this code block is not indented with 4 spaces it will break the ordered list

**Comment by eduardocardoso** [comment]
Everything seems fine now, thanks.

**Comment by realflash** [comment]
The last thing I can't fix is that I can't get the code blocks indented in the README like you somehow have. I've pasted directly from the raw version working in your repo, but it makes no difference.

**Comment by eduardocardoso** [comment]
On your last commit it was indented correctly

**Comment by realflash** [comment]
OK weird. Just not showing right in my repo then.

**Pull request merged**